includes/payment-tokens/class-wc-payment-token-cc.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * WooCommerce Credit Card Payment Token
+ *
+ * Representation of a payment token for credit cards
+ *
+ * @class 		WC_Payment_Token_CC
+ * @since		2.6.0
+ * @category 	PaymentTokens
+ * @package 	WooCommerce/PaymentTokens
+ * @author		WooThemes
+ */
+class WC_Payment_Token_CC extends WC_Payment_Token {
```

Comment:
This could extend from `WC_Payment_Token_eCheck` to save defining `get_last4()` and `set_last4()` in both classes (and even validating the last 4 only once as you could call `parent::validate()` to validate them in here instead). :smile: